corehq/apps/translations/validator.py

```diff
@@ -158,10 +158,15 @@ def _compare_sheet(self, module_or_form_id, uploaded_rows, for_type):
         :return: diff generated by ghdiff or None
         """
         columns_to_compare = COLUMNS_TO_COMPARE[for_type] + self.lang_cols_to_compare
-        header_indices = {
-            column_name: self._get_current_header_index(module_or_form_id, column_name)
-            for column_name in columns_to_compare
-        }
+        try:
+            header_indices = {
+                column_name: self._get_current_header_index(module_or_form_id, column_name)
+                for column_name in columns_to_compare
+            }
+        except KeyError:
+            raise BulkAppTranslationsException(_(
+                "No {} with Id '{}' is present in the Application.""Please verify your translations file."
```

Comment:
Doing the "" in separate lines will also work..I am surprised this didn't give you a line length warning.

Reply:
It's actually within the limits 😅

Reply:
cool. Also we are trying to use format strings, they are nice. Check [these](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) out.

Reply:
I tried using them in the code.
Thanks 😄